modules/caddyhttp/fileserver/matcher.go

```diff
@@ -298,20 +308,20 @@ func strictFileExists(file string) (os.FileInfo, bool) {
 // firstSplit returns the first result where the path
 // can be split in two by a value in m.SplitPath. The
 // result is the first piece of the path that ends with
-// in the split value. Returns the path as-is if the
-// path cannot be split.
-func (m MatchFile) firstSplit(path string) string {
+// in the split value, and the remainder. Returns the
+// path as-is if the path cannot be split.
+func (m MatchFile) firstSplit(path string) (string, string) {
```

Comment:
This is kinda awkward sounding, maybe: "The return values are the first piece of the path that ends with the split substring and the remainder. If the path cannot be split, the path is returned as-is (with no remainder)."